.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+// npx stylelint "**/*.{css,scss}"
+
+{
+  "env": { "browser": true, "es6": true, "jest": true },
+  "parserOptions": { "ecmaVersion": 2018, "sourceType": "module" },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off"
+  }
+}
```

Comment:
please get the original content of this file from [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)

and modify yours